Scripts/sourceclear.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+if [ "$RUN_SOURCECLEAR" = "true" ]; then
+    echo "RUN_SOURCECLEAR is set, running sourceclear"
+    curl -sSL https://download.sourceclear.com/ci.sh |  bash
+else
+    echo "RUN_SOURCECLEAR it not set, skipping"
```

Comment:
TYPO "it" SHOULD BE "is"
Additionally, at run time, if the "else" branch gets excercised we may know
from the bash code what is being skipped, but the console log will just be
printing "skipping", instead of a perhaps more helpful "skipping sourceclear"
message.